pom.xml

```diff
@@ -90,6 +90,9 @@
               <include>**/*TestCase.java</include>
               <include>**/*Tests.java</include>
             </includes>
+              <excludes>
+                  <exclude>ViewToolsTests.java</exclude>
```

Comment:
Where else are we running this test now?

Reply:
We need to deploy [showcase](https://github.com/atlassian-forks/velocity-tools/tree/2.0.x/examples) app after that we can test this, It was being done from ant script earlier